_rest_config.php

```diff
@@ -153,6 +153,24 @@ function destroySession()
             );
         }
     }
+
+    static function getPostData($data)
+    {
+        if (count($_POST)) {
+            return $_POST;
+        } elseif ($post_data = file_get_contents('php://input')) {
+            if ($post_json = json_decode($post_data, true)) {
+                return $post_json;
+            } else {
+                parse_str($post_data, $post_variables);
+                if (count($post_variables)) {
+                    return $post_variables;
+                }
+            }
+        }
+            // Not able to retrieve POST data, return FALSE.
+            return false;
```

Comment:
indentation off in above 2 lines (yes, that is likely only going to be my only input on this review :) )